docs/pages/blog/introducing-mui-base.md

```diff
@@ -0,0 +1,96 @@
+---
+title: 'Introducing MUI Base: the headless alternative to Material UI'
+description: 'Introduction to a new library of unstyled components and hooks: MUI Base'
```

Comment:
Let's make this a real pithy one-liner to drive home the main value proposition. Maybe something like this?

```suggestion
description: 'The MUI Base component library gives you complete control over the look and feel of your app.'
```